### ISSUE TITLE
make sync failing on untracked Android.mk

### ISSUE THREAD

**Issue by benfrancis** [andreasgal/B2G#184]
Earlier today fabrice reported a problem with make sync returning these errors http://pastebin.mozilla.org/1486325

He says it's working again for him now, but when I run make sync I get:

git pull origin master
From https://github.com/andreasgal/B2G
- branch            master     -> FETCH_HEAD
  Updating 976666d..be14c34
  error: Untracked working tree file 'glue/gonk/dalvik/libnativehelper/Android.mk' would be overwritten by merge.  Aborting
  make: **\* [sync] Error 1

**Comment by jlebar** [comment]
I just got that, too.  I deleted the files and then make sync worked.  But I dunno if those files will be re-created after my next build.  I'll find out in a few minutes, I guess.

**Comment by jlebar** [comment]
Yeah, deleting them worked fine.  The pull re-created those files.

**Comment by michaelwu** [comment]
Yeah a rm -rf glue/gonk/dalvik is recommended here. I'm not sure there's any way to make git behave better.

**Comment by timdream** [comment]
I got this too. Will try `rm -rf glue/gonk/dalvik`

**Comment by ThinkerYzu** [comment]
|make mrproper| or |git clean -dfx && git reset --hard|.
